Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml.cs

```diff
@@ -55,6 +65,17 @@ private void UpdateBadge(TMBadge tMBadge)
         UpdateShape(tMBadge);
         UpdateLabelOnSize(tMBadge);
     }
+    private static void OnContentPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+    {
+        if (bindable is TMBadge contentView && newValue != null)
```

Comment:
Shouldn't we set the values to null in the `else` condition?

Reply:
When content is removed the OnContentPropertyChanged() method is not called, hence the same logic can stand

Reply:
Does your comment remains the same after our discussion on Friday?